README.md

```diff
@@ -49,6 +49,13 @@ apt 'com.ryanharter.auto.value:auto-value-gson:0.2.2'
 
 (Using the [android-apt](https://bitbucket.org/hvisser/android-apt) plugin)
 
+In Order for the latest update to work (where this documentation is referring to) you need to have the snapshot
```

Comment:
Please remove the Jitpack reference.  There will always be certain features that are only available in snapshot builds, but we can't start adding (and updating) jitpack commit specific dependencies.